src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2114,25 +2114,75 @@ namespace UnitarySystems {
                     this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
                 }
             } else {
-                if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed ||
-                    this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingAirToAirVariableSpeed) {
-                    if (this->m_CoolCoilExists && this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed) {
```

Comment:
This check only happened for VS DX coil types. This change opens up the ability to set no load air flow rate for all other multi/variable-speed coils. The user can now choose how to set no load air flow rate by choosing either the low or high speed air flow rate. I hope I got all the coil types here that can have multiple air flow rates.